### PR TITLE
[157] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by HalimahOladimeji**
Laptop view
Screenshot 2020-12-04 010835
Screenshot 2020-12-04 145716
Tablet view
Screenshot 2020-12-04 150525
Screenshot 2020-12-04 150611
Mobile view
Screenshot 2020-12-04 154837
Screenshot 2020-12-04 154554

### CODE REVIEW

**Comment by Ireti203** [comment]
Weldone @HalimahOladimeji The input field on the footer for the mobile view looks very small. You should adjust it.

**Comment by HalimahOladimeji** [comment]
I have corrected it.

File: submissions/CSS/contactus.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <nav class="logo">
+        <img src="images/logo.png" alt="log" style="width: 75px; height: 50px;">
+        <ul class="headerc">
+            <li><a href="../submissions/CSS/index.html">Product Page</a></li>
+            <li><a href="../submissions/CSS/contactus.html">Contact Us</a></li>
+            <li><a href="#">Sign-up</a></li>
+        </ul>
+    </nav>
+    <div class="phone1">
+        <div class="firstc">
```
Comment (Ireti203) [inline submissions/CSS/contactus.html:22]: Kindly change this class name to a more descriptive name

**Comment by HalimahOladimeji** [comment]
Done.

File: submissions/CSS/contactus.css [hunk 2]
```diff
@@ -0,0 +1,228 @@
+body {
```
Comment (adaugochi) [inline submissions/CSS/contactus.css:1]: Kindly move this `CSS` folder and its contents inside your name folder. It shouldn't be inside the `submissions` folder

File: submissions/CSS/contactus.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/CSS/contactus.html:1]: This should be inside `html-forms` folder

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by HalimahOladimeji** [comment]
I've changed the locations of the files.

**Pull request closed without merging**